CHANGELOG.md

```diff
@@ -17,7 +17,7 @@ _No breaking changes!_
 
 **Gem enhancements:**
 
-- Reduced gem size from 173 kB compressed (312 kB unpacked) to 20 kB (68 kB unpacked) [[#345](https://github.com/panorama-ed/memo_wise/pull/345)]
+- Reduced gem size from 173 kB compressed (312 kB unpacked) to 18.5 kB (68 kB unpacked) [[#345](https://github.com/panorama-ed/memo_wise/pull/345)]
```

Comment:
Not sure why this was different on my machine, but 173 kB and 18.5 kB are the sizes [officially reported by rubygems.org](https://rubygems.org/gems/memo_wise), and 312 kB and 68 kB are the sizes I get when I download the gem from rubygems.org and then run `du -sh /path/to/.gem/ruby/3.3.3/gems/memo_wise-1.9.0` (or `-1.10.0`).